src/main/java/org/gradle/profiler/BenchmarkResultCollector.java

```diff
@@ -135,7 +139,7 @@ private double getPValue(int sample) {
             if (a.length == 0 || b.length == 0) {
                 return 1;
             }
-            return new MannWhitneyUTest().mannWhitneyUTest(a, b);
+            return (1 - new MannWhitneyUTest().mannWhitneyUTest(a, b)) * 100;
```

Comment:
Are we doing this to match how we calculate it in `gradle/gradle`?

Reply:
Yes, that gives the confidence which we show in the reports.